src/System.Text.Json/src/System/Text/Json/JsonUtf8Reader.cs

```diff
@@ -0,0 +1,1274 @@
+                    return SkipMultiLineComment(localCopy.Slice(1), out _);
+                else
+                    ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, marker);
+            }
+
+            if (IsLastSpan)
+                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, JsonConstants.Solidus);
+            else
+                return false;
+
+            return true;
+        }
+
+        private bool SkipSingleLineComment(ReadOnlySpan<byte> localCopy, out int idx)
+        {
+            //TODO: Match Json.NET's end of comment semantics
+            idx = localCopy.IndexOf(JsonConstants.LineFeed);
+            if (idx == -1)
+            {
+                if (IsLastSpan)
+                {
+                    idx = localCopy.Length;
+                    // Assume everything on this line is a comment and there is no more data.
+                    _lineBytePosition += 2 + localCopy.Length;
+                    goto Done;
+                }
+                else
+                    return false;
+            }
+
+            _consumed++;
+            _lineBytePosition = 0;
+            _lineNumber++;
+        Done:
+            _consumed += 2 + idx;
+            return true;
+        }
+
+        private bool SkipMultiLineComment(ReadOnlySpan<byte> localCopy, out int idx)
+        {
+            idx = 0;
+            while (true)
+            {
+                int foundIdx = localCopy.Slice(idx).IndexOf(JsonConstants.Solidus);
+                if (foundIdx == -1)
+                {
+                    if (IsLastSpan)
+                        ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.EndOfCommentNotFound);
+                    else
+                        return false;
+                }
+                if (foundIdx != 0 && localCopy[foundIdx + idx - 1] == '*')
+                {
+                    idx += foundIdx;
+                    break;
+                }
+                idx += foundIdx + 1;
+            }
+
+            Debug.Assert(idx >= 1);
+            _consumed += 3 + idx;
+
+            (int newLines, int newLineIndex) = JsonReaderHelper.CountNewLines(localCopy.Slice(0, idx - 1));
+            _lineNumber += newLines;
+            if (newLineIndex != -1)
+                _lineBytePosition = idx - newLineIndex;
+            else
+                _lineBytePosition += 4 + idx - 1;
+            return true;
+        }
+
+        private bool ConsumeComment()
+        {
+            //Create local copy to avoid bounds checks.
+            ReadOnlySpan<byte> localCopy = _buffer.Slice(_consumed + 1);
+
+            if (localCopy.Length > 0)
+            {
+                byte marker = localCopy[0];
+                if (marker == JsonConstants.Solidus)
+                    return ConsumeSingleLineComment(localCopy.Slice(1));
+                else if (marker == '*')
```

Comment:
Why do we have some tokens like '/' on JsonConstants but others like '*' not?

Reply:
We do, I just forgot to use it since I added it recently:
https://github.com/dotnet/corefx/pull/33216/files#diff-f71b8b25774c29bdc1c44926f32b7ecbR24